CSS/README.md

```diff
@@ -102,55 +102,115 @@
   > - Pros: Nested tags, Mixins, Importing other styles in, Modularity
   > - Cons: Nested tags are hard to read after a certain point, Have to use build tools to compile, Easy to abuse (@extend in sass)
 
-- [ ] Are you familiar with styling SVG?
-- [ ] How do you optimize your webpages for print?
-- [ ] What are some of the "gotchas" for writing efficient CSS?
-- [ ] How would you implement a web design comp that uses non-standard fonts?
-- [ ] Explain how a browser determines what elements match a CSS selector.
-- [ ] Describe pseudo-elements and discuss what they are used for.
-- [ ] Explain your understanding of the box model and how you would tell the browser in CSS to render your layout in different box models.
-- [ ] What does `* { box-sizing: border-box; }` do? What are its advantages?
-- [ ] The 'C' in CSS stands for Cascading. How is priority determined in assigning styles (a few examples)? How can you use this system to your advantage?
-- [ ] How is responsive design different from adaptive design?
-- [ ] Have you ever worked with retina graphics? If so, when and what techniques did you use?
+- [x] Considerations for printing webpages?
+  
+  > - Specify a Print Style Sheet `<link rel="stylesheet" href="print.css" media="print" />`
+  > - Hide Extraneous Elements `.navigation, .footer { display: none;}` 
+  > - Bump Up Font Sizes and Line-Heights `body { font-size: 120%; line-height: 130%; }`
+  > - Move to Serif Fonts `body { font-family: Georgia, "Times New Roman", Times, serif; }`
+  > - Display the destination of links & make them standout with different color `a:link:after { content: " (" attr(href) ") "; }`
+  > - Enlarge content area `#content { width: 100%; margin: 0; float: none; }`
+
+- [x] Explain how a browser determines what elements match a CSS selector.
+
+  > - CSS Selectors are matched by browser engines from right to left consider `#menu ul li a { color: #00f; }` The browser first checks for `a`, then `li`, then `ul`, and then `#menu`.
+  > - Check matching elements for the key(right-most) selector
+  > - Check if the elements are matching parents for the next selectors
+
+- [x] How would you implement a web design comp that uses non-standard fonts?
+  
+  > - Google web font is a best alternative to non-standard fonts
+  > - If it's a non-standard and custom font then we can use `@font-face` to render it.
+
+- [x] What does `* { box-sizing: border-box; }` do? What are its advantages?
+
+  > - Apply a natural box layout model to all elements, but allowing components to change. 
+  > - If you set an element width to 300px and if you apply 10px padding then the rendered element on browser would take 300 + leftmargin 20 + rightmargin 20 = total of 340px in width. If you give `box-sizing: border-box` the padding is applied right inside your box model and rendered width would be 300px irrespective of padding / border used.
```

Comment:
This explain the above. May me mentioned the `content section in the element shrinks` in order to not take more the `300px` width